Dockerfile

```diff
@@ -11,6 +11,10 @@ ENV JAVA_HOME /usr/lib/jvm/java-7-openjdk-amd64
 
 ENV MESOS_NATIVE_JAVA_LIBRARY /usr/lib/libmesos.so
 
+ARG MESOS_RELEASE
+ARG STORM_RELEASE
+ARG MIRROR
```

Comment:
Ah good point, I was thinking people would only invoke build via the build-release.sh script.